kubectl-plugin/install/installer.go

```diff
@@ -24,6 +24,11 @@ You can get more details about the ks-installer from https://github.com/kubesphe
 		RunE:    opt.runE,
 	}
 
+	commander := Commander{}
+	if err := commander.execCommand("docker", "ps"); err != nil {
+		return
+	}
+
```

Comment:
There is no necessary to check the status of the Docker service during the command initial process.

Reply:
Yes, I have verified that it is feasible, so this PR should be closed

Reply:
ok